### ISSUE TITLE
[Bug Report] el-input regression when using v-model.number

### ISSUE THREAD

**Issue by skolos** [ElemeFE/element#18003]
### Element UI version
0.1.0

### OS/Browsers version
Windows/Firefox 70.0.1, Chrome 78.0.3904.70

### Vue version
2.6.10

### Reproduction Link
https://codepen.io/skolos/pen/bGGKKdX

### Steps to reproduce
1. Try typing into input box: 1.1
It will not allow typing decimal period and will end up typing 11

2. If change element-ui version to 2.4.11 (i.e. using "//unpkg.com/element-ui@2.4.11/lib/index.js" link in HTML part), there is no issue (i.e. will end up typing 1.1)

3. If change element-ui version to anything from 2.5.0 and up, the issue is present.

### What is Expected?
Will be able to type 1.1

### What is actually happening?
Cannot type 1.1, instead getting 11.

Looks like the regression is caused by this PR: https://github.com/ElemeFE/element/pull/13471

**Comment by Erbenos** [comment]
Please read docs more carefully.

See: https://element.eleme.io/#/en-US/component/input#input

[type] attribute on el-input has accepted values of: text, textarea and other native input types

The `age` in your codepen doesn't make sense. Changing that to `number` fixes the issue.

Can someone close this?

**Comment by skolos** [comment]
The codepen that I referenced comes directly from element-ui's
documentation. You can see the same behavior in example shown in the
documentation:
https://element.eleme.io/#/en-US/component/form#number-validate

So please, don't close this until the behavior makes sense and/or example
in documentation is fixed.

Thanks

**Comment by Erbenos** [comment]
You are totally right this is an issue. I just provided a workaround. Should be fixed in either case.

**Comment by skolos** [comment]
The issue is a regression and still is not resolved.

**Comment by jonus1993** [comment]
Change type form age on number:
`<el-input type="number" v-model.number="inputVal"></el-input>`
